lib/events.js

```diff
@@ -88,8 +113,47 @@ EventEmitter.init = function() {
   }
 
   this._maxListeners = this._maxListeners || undefined;
+
+  if (opts && opts.captureRejections) {
```

Comment:
I'd prefer strict typing on this. e.g.

like...
```js
const { captureRejections = false } = { ...opts };

if (typeof captureRejections !== 'boolean') {
  throw new ERR_INVALID_ARG_TYPE('options.captureRejections',
                                 'boolean', captureRejections);
}
```

Reply:
would still rather this use strict typing

Reply:
Using that type of syntax is slower, and we create a lot of EE. I'll make it stricter anyway.